lib/LedgerSMB/PriceMatrix.pm

```diff
@@ -94,6 +94,7 @@ Updates $ref with the price matrix outcomes given $transdate and $form.
 
 sub price_matrix {
     my ( $pmh, $ref, $transdate, $decimalplaces, $form, $myconfig) = @_;
+    return if $form->{id};
```

Comment:
@einhverfr I know we discussed this, but what happens if/when a saved invoice gets edited before it gets posted?